### PR TITLE
Link to network monitor

### LINKED ISSUE #487: Link to network monitor
Include a link to the EOSCR network monitor from sidebar menu

[EOS Network Monitor](https://mainnet.eosio.cr/)

### PR DESCRIPTION
**Pull request by AngeloCG97**
### Link to network monitor 

Resolve #487

### Steps to test

1. Change componente link
1. Add EOS Network Monitor link
2. Add env var

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [ ] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [ ] I have taken into consideration any impact to site performance.

### CODE REVIEW

File: services/frontend/src/routes/index.js [hunk 1]
```diff
@@ -8,54 +8,72 @@ import ProxyProfile from './proxies/proxy-profile'
 import Help from './help'
 import TermsOfUse from './termsOfUse'
 import About from './about'
+import config from '../config'
+
+console.log('config.networkMonitor: ', config)
```
Comment (xavier506) [inline services/frontend/src/routes/index.js:13]: do we need this console log?

File: docker-compose.yml [hunk 2]
```diff
@@ -119,5 +119,6 @@ services:
       REACT_APP_EOS_API_PROTOCOL: "${REACT_APP_EOS_API_PROTOCOL}"
       REACT_APP_EOS_CHAIN_ID: "${REACT_APP_EOS_CHAIN_ID}"
       REACT_APP_BLOCK_EXPLORER: "${REACT_APP_BLOCK_EXPLORER}"
+      REACT_APP_NETWORK_MONITOR_URL: "${REACT_APP_NETWORK_MONITOR_URL}"
```
Comment (leisterfrancisco) [inline docker-compose.yml:122]: Add this to variable to `docker-compose.staging.yml` please

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (AngeloCG97) [reply docker-compose.yml:122]: This file need more atention, i create new issue for this topic https://github.com/eoscostarica/eos-rate/projects/1#card-65477072

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**